tests/fixture_plugins/demultiplex_fixtures/sample_fixtures.py

```diff
@@ -138,6 +200,28 @@ def seven_canonical_flow_cells_selected_sample_ids(
     ]
 
 
+@pytest.fixture
+def seven_canonical_sequencing_runs_selected_case_ids(
+    selected_hiseq_x_single_index_case_ids: list[str],
+    selected_hiseq_x_dual_index_case_ids: list[str],
+    selected_hiseq_2500_dual_index_case_ids: list[str],
+    selected_hiseq_2500_custom_index_case_ids: list[str],
+    selected_novaseq_6000_pre_1_5_kits_case_ids: list[str],
+    selected_novaseq_6000_post_1_5_kits_case_ids: list[str],
+    selected_novaseq_x_case_ids: list[str],
+) -> list[list[str]]:
+    """Return a list of lists containing the selected case ids for each canonical flow cell."""
+    return [
+        selected_hiseq_x_single_index_case_ids,
+        selected_hiseq_x_dual_index_case_ids,
+        selected_hiseq_2500_dual_index_case_ids,
+        selected_hiseq_2500_custom_index_case_ids,
+        selected_novaseq_6000_pre_1_5_kits_case_ids,
+        selected_novaseq_6000_post_1_5_kits_case_ids,
+        selected_novaseq_x_case_ids,
+    ]
```

Comment:
I feel like linking sequencing runs to cases is very sketchy and we should only ever do it in the NIPT flow (since there are no plans to change it). Can we limit the a NIPT-case with one FC and one with two? (and skip association cases to our fancy canonical runs)